spec/github_pages_health_check/domain_spec.rb

```diff
@@ -836,13 +860,27 @@
     context "A records pointed to old IPs" do
       let(:ip) { "192.30.252.153" }
       before(:each) { allow(subject).to receive(:dns) { [a_packet] } }
+      before(:each) { allow(subject.send(:caa)).to receive(:query) { [a_packet] } }
```

Comment:
This is required so that we return the `A` when no `CAA` exists, imitating the behaviour seen in the real world. This hasn't been a issue until now, as the existing checks use a `domain` of `pages.github.com`, so we get back entries from the real world here. It is only now that I have started using fake names that this has become a problem.